sklearn/linear_model/tests/test_base.py

```diff
@@ -165,6 +211,32 @@ def test_linear_regression_sparse(random_state=0):
         assert_array_almost_equal(ols.predict(X) - y.ravel(), 0)
 
 
+@pytest.mark.parametrize(
+    'normalize, n_warnings, warning',
+    [(True, 1, FutureWarning),
+     (False, 1, FutureWarning),
+     ("deprecated", 0, None)]
+)
+# FIXME remove test in 1.4.0
+def test_assure_warning_when_normalize(normalize, n_warnings, warning):
+    # check that we issue a FutureWarning when normalize was set
+    rng = check_random_state(0)
+    n_samples = 200
+    n_features = 2
+    X = rng.randn(n_samples, n_features)
+    X[X < 0.1] = 0.
```

Comment:
```suggestion
    X[X < 0.1] = 0.0
```

Reply:
here again this is subjective style. 0. is perfectly fine to me.